Sources/TuistGenerator/Templates/PlistsTemplate.swift

```diff
@@ -44,7 +44,7 @@ extension SynthesizedResourceInterfaceTemplates {
     {% endmacro %}
     {% macro valueBlock value metadata %}
       {%- if metadata.type == "String" -%}
-        "{{ value }}"
+        "{{ value|replace: "\\n", " ", "regex" }}"
```

Comment:
@danyf90
`replace: "\\n", "\n", "regex"` 
I have also tried this method, but it doesn't seem to work as intended.